test/src/practice/quiz/widgets/quiz_app_bar_test.dart

```diff
@@ -0,0 +1,78 @@
+import "package:flutter/material.dart";
+import "package:flutter_test/flutter_test.dart";
+import "package:kana_to_kanji/src/core/widgets/rounded_linear_progress_indicator.dart";
+import "package:kana_to_kanji/src/practice/quiz/widgets/quiz_app_bar.dart";
+
+import "../../../../helpers.dart";
+
+bool onClosePressedCalled = false;
+bool onSkipPressedCalled = false;
+
+void onClosePressed() {
+  onClosePressedCalled = true;
+}
+
+Future<void> onSkipPressed() async {
+  onSkipPressedCalled = true;
+}
+
+void main() {
+  group("QuizAppBar", () {
+    const progressBarValue = 22.6;
+    testWidgets("Check display", (WidgetTester tester) async {
```

Comment:
To unify with the rest of the code

```suggestion
    testWidgets("UI", (WidgetTester tester) async {
```